lib/Form/Checkbox/Group.js

```diff
@@ -19,10 +20,12 @@ export default class Group extends Component {
   constructor(props) {
     super(props);
     const selected = props.choices.filter(c => c.checked);
-    this.state = { choices: props.choices, selected };
+    this.state = { choices: props.choices, selected, highlightHover: false };
```

Comment:
What is the `highlightHover` state used for? I've taken a look elsewhere but I didn't wan't to assume something incorrectly 😄

Reply:
if its true it will show the hover style for the whole checkbox list, rather than just the individual checkbox option